### PR TITLE
Add zipcode->states

### PR DESCRIPTION
**Pull request by ambroisep**

### CODE REVIEW

File: src/zipcode/core.clj [hunk 1]
```diff
@@ -1,9 +1,27 @@
...
+            [clojure.edn :as edn]
+            [clojure.string :as string]))
 
-(def zipcode-db (into {} (csv/parse-csv (slurp (io/resource "zipcode-database.csv")))))
+(def zipcode-db
+  ;; source is Google BigQuery `bigquery-public-data.utility_us.zipcode_area`
+  (->> (edn/read-string (slurp (io/resource "zipcode-google-bigquery-database-20170713.edn")))
+       (mapv (fn [{:keys [zipcode state_code]}]
+               [(remove-preceeding-zeros zipcode)
+                (set (string/split state_code #", "))]))
```
Comment (mikekap) [inline src/zipcode/core.clj]: nit: `(->> (string/split state_code #",") (map string/trim) set)`

File: src/zipcode/core.clj [hunk 2]
```diff
@@ -1,9 +1,27 @@
 (ns zipcode.core
   (:require [clojure.java.io :as io]
-            [clojure-csv.core :as csv]))
+            [clojure.edn :as edn]
+            [clojure.string :as string]))
 
-(def zipcode-db (into {} (csv/parse-csv (slurp (io/resource "zipcode-database.csv")))))
+(def zipcode-db
+  ;; source is Google BigQuery `bigquery-public-data.utility_us.zipcode_area`
```
Comment (mikekap) [inline src/zipcode/core.clj]: do you want to include the SQL query you used?

File: test/zipcode/core_test.clj [hunk 3]
```diff
@@ -2,6 +2,26 @@
   (:require [clojure.test :refer :all]
             [zipcode.core :refer :all]))
 
-(deftest a-test
-  (testing "FIXME, I fail."
-    (is (= 0 1))))
```
Comment (mikekap) [inline test/zipcode/core_test.clj:7]: lal

**Review by mikekap — APPROVED** [review]

File: src/zipcode/core.clj (hunk 2 above)
Comment (ambroisep) [reply src/zipcode/core.clj]: good call
will do

**Pull request merged**